uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/models/entitlement-model_test.js

```diff
@@ -0,0 +1,84 @@
+define([
```

Comment:
Why is this test in a `test/my-leave` folder? The model does not belong to `my-leave`, it's a shared model